api/inventorypb/agents.proto

```diff
@@ -1327,208 +1328,328 @@ service Agents {
   rpc AddRDSExporter(AddRDSExporterRequest) returns (AddRDSExporterResponse) {
     option (google.api.http) = {
       post: "/v1/inventory/Agents/AddRDSExporter"
       body: "*"
     };
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+      summary: "Add RDS Exporter"
+      description: "Adds rds_exporter Agent."
+    };
   }
   // ChangeRDSExporter changes rds_exporter Agent.
   rpc ChangeRDSExporter(ChangeRDSExporterRequest) returns (ChangeRDSExporterResponse) {
     option (google.api.http) = {
       post: "/v1/inventory/Agents/ChangeRDSExporter"
       body: "*"
     };
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+      summary: "Change RDS Exporter"
+      description: "Changes rds_exporter Agent."
+    };
   }
   // AddExternalExporter adds external_exporter Agent.
   rpc AddExternalExporter(AddExternalExporterRequest) returns (AddExternalExporterResponse) {
     option (google.api.http) = {
       post: "/v1/inventory/Agents/AddExternalExporter"
       body: "*"
     };
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+      summary: "Add External Exporter"
+      description: "Adds external_exporter Agent."
+    };
   }
   // ChangeExternalExporter changes external_exporter Agent.
   rpc ChangeExternalExporter(ChangeExternalExporterRequest) returns (ChangeExternalExporterResponse) {
     option (google.api.http) = {
       post: "/v1/inventory/Agents/ChangeExternalExporter"
       body: "*"
     };
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+      summary: "Change External Exporter"
+      description: "Changes external_exporter Agent."
+    };
   }
   // AddAzureDatabaseExporter adds azure_database_exporter Agent.
   rpc AddAzureDatabaseExporter(AddAzureDatabaseExporterRequest) returns (AddAzureDatabaseExporterResponse) {
     option (google.api.http) = {
       post: "/v1/inventory/Agents/AddAzureDatabaseExporter"
       body: "*"
     };
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+      summary: "Add Azure Database Exporter"
+      description: "Adds azure_database_exporter Agent."
+    };
   }
-  // ChangeAzureDatabaseExporter changes_database azure_database_exporter Agent.
+  // ChangeAzureDatabaseExporter changes azure_database_exporter Agent.
   rpc ChangeAzureDatabaseExporter(ChangeAzureDatabaseExporterRequest) returns (ChangeAzureDatabaseExporterResponse) {
     option (google.api.http) = {
       post: "/v1/inventory/Agents/ChangeAzureDatabaseExporter"
       body: "*"
     };
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+      summary: "Change Azure Database Exporter"
+      description: "Changes azure_database_exporter Agent."
+    };
   }
   // RemoveAgent removes Agent.
   rpc RemoveAgent(RemoveAgentRequest) returns (RemoveAgentResponse) {
     option (google.api.http) = {
       post: "/v1/inventory/Agents/Remove"
```

Comment:
```suggestion
      post: "/v1/inventory/Agents/RemoveAgent"
```

Reply:
It's part of API definition, we can't change it because of backward compatibility